### ISSUE TITLE
MrMime ?

### ISSUE THREAD

**Issue by dinosaure** [fxfactorial/ocaml-mailing-list#1]
May be it's a good PoC for [MrMime](https://github.com/oklm-wsh/MrMime) instead a JSON file ?

**Comment by fxfactorial** [comment]
What's mrmime? Send a pr poc?

**Comment by dinosaure** [comment]
A yes, I forgot to write a presentation of this project :) ! It's a parser of email - for the moment, we can parse 18k emails without any exception. So, I work on the API but the core is stable. May be it's useful to use MrMime inside your project to extract the email's information.

**Comment by fxfactorial** [comment]
Ah, I'm more focused on generating the HTML meant to be used for an email rather than parsing emails. Can you generate an email as an API? Perhaps also send?

**Comment by dinosaure** [comment]
Yes, it's the goal of MrMime, a good API to read and write an email :)

**Comment by fxfactorial** [comment]
Okay, cool although not sure how I could use it now. but yea, feel free to add usage if possible
